src/util/util.c

```diff
@@ -608,6 +608,9 @@ bool is_host_in_domain(const char *host, const char *domain)
 }
 
 /* addr is in network order for both IPv4 and IPv6 versions */
+#ifndef IN_LOOPBACK  /* from <linux/in.h> */
```

Comment:
Hi,

thanks, just a nitpick, can you move the definition so that the comment is again in front of `bool check_ipv4_addr()`? It currently looks like the comment relates to the definition.

bye,
Sumit